build/gen_stub.php

```diff
@@ -1650,13 +1654,21 @@ static function (Expr $expr) use ($allConstInfos, &$isUnknownConstValue) {
                     if ($constType) {
                         if ($constType->isBool()) {
                             return true;
-                        } elseif ($constType->isInt()) {
+                        }
+
+                        if ($constType->isInt()) {
                             return 1;
-                        } elseif ($constType->isFloat()) {
+                        }
+
+                        if ($constType->isFloat()) {
                             return M_PI;
-                        } elseif ($constType->isString()) {
+                        }
+
+                        if ($constType->isString()) {
                             return $const->name;
-                        } elseif ($constType->isArray()) {
+                        }
+
+                        if ($constType->isArray()) {
                             return [];
                         }
```

Comment:
I'm not sure this is an improvement. I think the `elseif` makes it clearer, that at most one of these apply (because it effectively is a switch).

Reply:
I agree. IMO this falls into the category of code style. We should avoid PRs that switch between code styles that are not objectively better, unless we have guidelines that prefers one over the other.

Reply:
I'd join Tim and Ilija with my opinion regarding coding style. Nikita and I wrote the vast majority of this file, with slightly different coding style choices, and despite this, cooperation worked well so far. That's why neither I really mind coding style here, especially because it's just a few thousands line code. The situation is different at my daytime work, where a larger group of people maintain hundreds of thousands of lines of code. There we have a very thorough coding style, (although using early returns is only recommended, but regulated in it either).

Reply:
Absolutely makes sense, thank you!